Rakefile

```diff
@@ -21,6 +21,11 @@ task :style_correct do
   system("swiftlint", "autocorrect")
 end
 
+desc("Swift format check")
+task :swift_format do
+  systemWithErrorMessage("swiftformat", "--lint", ".", "Swift formatting errors encountered, please run following terminal command `rake style_correct` locally from your tuist root folder to correct it and commit the change")
```

Comment:
Ruby's convention for method names is using snake case: `system_with_error_message`. Also, I don't think you need to print a custom error. The system command should print the `swiftformat` logs that should be descriptive enough. You can turn that line into:

```rb
Kernel.system("swiftformat", "--lint", ".") || abort 
```